hilla-jandex/pom.xml

```diff
@@ -42,11 +42,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>com.github.mcollovati</groupId>
-            <artifactId>hilla-shaded-deps</artifactId>
-            <version>${project.version}</version>
-        </dependency>
```

Comment:
Deletion correct?

Reply:
Yes, it is intended. The dependency has been moved to the commons module